src/App.test.jsx

```diff
@@ -0,0 +1,78 @@
+import { fireEvent, render } from '@testing-library/react';
+
+import { tasks } from '../fixtures/tasks';
+
+import App from './App';
+
+describe('App Component', () => {
+  const taskTitle = '코드숨 과제하기';
+  const renderComponent = () => render((
+    <App />
+  ));
+
+  context('Todo Appender', () => {
+    it('할 일을 입력할 수 있다.', () => {
+      const { getByPlaceholderText } = renderComponent();
+      const input = getByPlaceholderText('할 일을 입력해 주세요');
+
+      fireEvent.change(input, {
+        target: {
+          value: taskTitle,
+        },
+      });
```

Comment:
```suggestion
      const { getByPlaceholderText } = renderComponent();

      const input = getByPlaceholderText('할 일을 입력해 주세요');
      fireEvent.change(input, {
        target: {
          value: taskTitle,
        },
      });
```

이렇게 묶어주는게 좀 더 읽기 좋을 것 같아요.

Reply:
제가 생각했을 땐, input 부분을 given 이라 생각했고, fireEvent 부분을 when 이라 생각했는데 
잘못된 접근 방식이었나요?

Reply:
잘못된 것은 없습니다. `input`은 이벤트를 발생시키는 엘리먼트 인것 같아요같아요